helm/cosmo/charts/router/templates/gateway.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if .Values.istioGateway -}}
+{{- if .Values.istioGateway.enabled -}}
+{{- $fullName := include "router.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "router.labels" . | nindent 4 }}
+  {{- with .Values.istioGateway.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    app: {{ .Values.istioGateway.gatewayName }}
+  servers:
+  {{- range .Values.istioGateway.hosts }}
```

Comment:
Please provide default values for all possible options in the values.yaml. If we add a`# -- ` comment to all values we can auto generate the docs. Don't forget to run `make docs` in the `helm` directory.